spec/sidekiq/throttled/fetch_spec.rb

```diff
@@ -50,6 +49,39 @@
     end
   end
 
+  shared_examples "provides bulk requeue API" do |scope|
+    before do
+      Sidekiq::Client.push_bulk({
+        "class" => WorkingClassHero,
+        "args"  => Array.new(3) { [1, 2, 3] }
+      })
```

Comment:
I think this can be simplified to:

```suggestion
      3.times { |i| WorkingClassHero.perform_async(i) }
```

But I'm totally fine either way.